README.md

```diff
@@ -1,12 +1,28 @@
 # Docker-based Jenkins quick start examples
 
-This is a experiment repository for GSoC 2023 docker based jenkins quick start examples project 
+* This is a experiment repository for GSoC 2023 docker based jenkins quick start examples project 
 
 ## Proposal
 
-You can find the proposal for this project [here](https://docs.google.com/document/d/1ZpPihadYqpAvR20rxZkTD2SVpf34E6YMzg6opU6yHAg/edit#heading=h.lntg56ljm653).
+* You can find the proposal for this project [here](https://docs.google.com/document/d/1ZpPihadYqpAvR20rxZkTD2SVpf34E6YMzg6opU6yHAg/edit#heading=h.lntg56ljm653).
 
 ## Links
 
 * Meeting notes for this project can be found [here](https://docs.google.com/document/d/1yij9OvM2_92My3vqjn6u8ABHjXcyy0a7O6oM30b6ctM/edit).
 * Project page can be found [here](https://www.jenkins.io/projects/gsoc/2023/projects/docker-compose-build/).
+
+## How to Run with Gitpod  
+* Gitpod is a cloud development environment for teams to efficiently and securely develop software.
+* Gitpod support many IDEs to choose from such as VScode, IntelliJ and many more.  
+
+ ### How to setup the repo in Gitpod?
```

Comment:
Thanks for the references.

This is more about description instead of instruction. It would be better if we had more detailed steps, just like writing a tutorial.

I can adapt and guess how to complete the setup, but unsure for the beginner.

Reply:
I'll add more detailed steps